backend/src/openarchiefbeheer/config/api/views.py

```diff
@@ -80,3 +82,48 @@ def get(self, request: Request, *args, **kwargs):
         config = OpenIDConnectConfig.get_solo()
         serializer = OIDCInfoSerializer(instance=config, context={"request": request})
         return Response(serializer.data)
+
+
+class HealthCheckView(APIView):
+    @extend_schema(
+        summary=_("Get health check"),
+        description=_(
+            "Returns whether everything that should have been configured was configured."
+        ),
+        tags=["Configuration"],
+        responses={
+            200: inline_serializer(
+                name="HealthCheckResponse",
+                fields={
+                    "success": serializers.BooleanField(
+                        help_text="Whether the configuration is complete."
+                    ),
+                    "errors": inline_serializer(
+                        name="HealthCheckError",
+                        many=True,
+                        fields={
+                            "model": serializers.CharField(
+                                help_text="The model that is not properly configured."
+                            ),
+                            "code": serializers.CharField(
+                                help_text="A code name for the error."
+                            ),
+                            "message": serializers.CharField(
+                                help_text="A human readable error message."
+                            ),
+                            "severity": serializers.CharField(
+                                help_text="Whether this is an error, warning or info."
+                            ),
+                            "field": serializers.CharField(
+                                help_text="The field of a model that is not properly configured. Can be empty.",
+                                required=False,
+                            ),
+                        },
+                    ),
+                },
+            ),
+        },
+    )
+    def get(self, request: Request, *args, **kwargs):
+        results = is_configuration_complete()
```

Comment:
This will simply answer with whether the system is good to go or not? I think we need detailed information on what checks have failed/passed. I assume the implementation shows a list of bullets indicating the status of each individual service.

Reply:
No it returns also all of the errors (for example the ones shown in the frontend PR).

All the possible errors that can be detected are shown in backend/src/openarchiefbeheer/config/health_checks.py from line 24